tests/post/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import requests
+
+from pytest_bdd import given
+from pytest_bdd import when
+
+from utils.helper import Inventory
+
+
+@pytest.fixture(scope="session")
+def inventory_obj(inventory):
+    return Inventory(filename=inventory)
+
+
+@pytest.fixture(scope="session")
+def kubectl_proxy(request, run_services, watcher_getter, kubeconfig):
+    def api_checker():
+        try:
+            response = requests.get('http://127.0.0.1:8001/')
+        except requests.exceptions.RequestException:
+            return False
+        return bool(response)
```

Comment:
This `bool(exception)` is still a bit too magical to me as to what it's supposed to represent (what's success, what's failure?). I'd prefer this to be more explicit.

Reply:
There is 3 cases: exception, returncode 2xx, other return code. That a way to tell difference between the last 2 items

Reply:
Then I would prefer something like
```python
return 200 <= response.returncode < 300
```
or whatever the correct attribute name is.